celery/worker/request.py

```diff
@@ -520,6 +520,11 @@ def _context(self):
         }, **embed or {})
         return Context(request)
 
+    @cached_property
+    def group_index(self):
+        # used by backend.on_chord_part_return to order return values in group
+        return self.request_dict['group_index']
```

Comment:
Perhaps it is better to use `dict.get` here too.

Reply:
Done.